docroot/themes/humsci/humsci_basic/src/scss/components/_views-field-field-hs-person.scss

```diff
@@ -0,0 +1,44 @@
+.views-field-field-hs-person {
+  &-title {
+    font-size: 1.125rem;
+    font-weight: hb-theme-font-weight(semibold);
+    line-height: 122%;
+    margin-bottom: 0.8125rem;
+  }
+
+  &-email {
+    word-break: break-all;
+
+    a span {
+      display: none; // hide email icon
+    }
+  }
+
+  &-interests {
+    margin-bottom: 1.75rem;
+
+    .views-label-field-hs-person-interests {
+      display: inline-block;
+      margin-bottom: 0.875rem;
+      color: $hb-color--black;
+    }
+  }
+
+  &-education {
+    margin-bottom: 1.75rem;
+
+    .views-label-field-hs-person-education {
+      display: inline-block;
+      margin-bottom: 0.875rem;
+      color: $hb-color--black;
+    }
+
+    li {
+      @include hb-colorful {
+        &::before {
+          background-color: hb-colorful-variation(tertiary);
+        }
+      }
+    }
+  }
+}
```

Comment:
These styles are won't get picked up if they use hb_structured card with different content. Can we pull out the styles that are specific to the structured card into their own file?